UnitTests/BraintreeShopperInsightsTests/BTShopperInsightsClient_Tests.swift

```diff
@@ -6,7 +6,8 @@ import XCTest
 
 class BTShopperInsightsClient_Tests: XCTestCase {
     
-    var mockAPIClient = MockAPIClient(authorization: "development_client_key")!
+    let clientToken = TestClientTokenFactory.token(withVersion: 3)
+    var mockAPIClient: MockAPIClient?
```

Comment:
We do this a lot in our tests so we don't have to unwrap the optional in each test case (similar to what we do for the `sut`)

```suggestion
    var mockAPIClient: MockAPIClient!
```